.github/pull_request_template.md

```diff
@@ -0,0 +1,35 @@
+PLEASE FOLLOW THE CHECKLIST BELOW WHEN CREATING A NEW PULL REQUEST. THE
+CHECKLIST IS FOR YOUR INFORMATION AN MUST BE REMOVED BEFORE SUBMITTING THE PULL
+REQUEST.
```

Comment:
Maybe we should move the actual checklist outside of the template? And only link to that from here?

Reply:
I consider it more usable when given directly. I do not need to follow another link and have all the information already available. I can start to write directly above this. Even if we add just a link, the user still needs to remove the guideline text and we would not make it simpler.

Reply:
But you have to delete the text, which I expect to not happen frequently. Not important though, so if you prefer this, let's do it.